Doc/library/ftplib.rst

```diff
@@ -33,8 +33,10 @@ Here's a sample session using the :mod:`ftplib` module::
    drwxr-sr-x    4 1176     1176         4096 Nov 17  2008 project
    drwxr-xr-x    3 1176     1176         4096 Oct 10  2012 tools
    '226 Directory send OK.'
-   >>> ftp.retrbinary('RETR README', open('README', 'wb').write)
+   >>> ftp.retrbinary('RETR README', open('README', 'wb').write)  # Download the file README
```

Comment:
Hi @pinguin999, thanks for improving the documentation!

It seems to me that the file you open here is not closed before doing the upload next line. Shouldn't you close it before opening it again and reading from it?

Maybe we should use context managers here, after all, it probably isn't great to encourage beginners to keep their files open.